Simplenote/src/androidTest/java/com/automattic/simplenote/models/NoteTest.java

```diff
@@ -18,91 +18,75 @@
 @SmallTest
 public class NoteTest {
 
-    private static final String TEST = "test";
+    private static final String CONTENT_PREVIEW = "consectetur adipisicing elit,"
+            + "sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+            + "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+            + "Duis aute irure dolor in reprehenderit in voluptate velit esse cil.";
+    private static final String CONTENT_TITLE = "Lorem ipsum dolor sit amet";
+    private static final String CONTENT = CONTENT_TITLE + "\n" + CONTENT_PREVIEW;
+    private static final String KEY = "test";
     private Note mNote;
 
     @Before
     public void setUp() throws Exception {
```

Comment:
```suggestion
    public void setUp() {
```

We can remove `throws JSONException` since it's unused.